config/routes.rb

```diff
@@ -6,7 +6,11 @@
 
   namespace :v1, defaults: { format: 'json' } do
     jsonapi_resources :groups
-    jsonapi_resources :bulletins
+
+    jsonapi_resources :bulletins do
+      get 'sign', on: :collection
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.